frontend/src/gamestate.js

```diff
@@ -154,7 +135,7 @@ class GameState extends EventEmitter {
     remove(zone, (c) => c.name === card.name);
     // add n land
     range(n).forEach(() => zone.push(card));
-    this.#landDistribution[zoneName][card.colorSign] = n;
+    this.#landDistribution[zoneName][card.colorIdentity[0]] = n;
```

Comment:
Are you sure the colorIdentity is always right ? @ZeldaZach could you tell us if there's no caveats with colorIdentity that may not represent the true colors of the card (like devoid or else) ?

Reply:
This is a piece I coded by pattern matching. Thinking :
- changed the basic land const, and noticed that had colorSign property
- searched the app and didn't find any other dependences on this variable other than this
- saw that generally cards being passed around by app don't have colorSign
- decided to use colorIdentity which is already on the cards im basic lands.js because **assumed those were the only cards we were looking for a color sign on**

Might need someone to check that's true. These #properties look mysterious to me

Reply:
the # just indicates that the property is private.